src/firebolt/async_db/_types.py

```diff
@@ -263,3 +258,23 @@ def process_token(token: Token) -> Token:
         )
 
     return formatted_sql
+
+
+def format_sql(query: str, parameters: Sequence[ParameterType]) -> str:
+    return format_statement(parse_sql(query)[0], parameters)
+
+
+def split_format_sql(
+    query: str, parameters: Sequence[Sequence[ParameterType]]
+) -> List[str]:
+    statements = parse_sql(query)
+    if not statements:
+        return [query]
+
+    if parameters:
+        if len(statements) > 1:
+            raise NotSupportedError(
+                "formatting multistatement queries is not supported"
+            )
+        return [format_statement(statements[0], paramset) for paramset in parameters]
+    return [str(st).strip().rstrip(";") for st in statements]
```

Comment:
Using strip is copied from sqparse `.split` implementation.
`rstrip` is used just for consistency, since some queries are parsed with ending ";" and some are without.